tests/config_test.go

```diff
@@ -195,11 +196,11 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				secretName = "secret-" + uuid.NewString()
 				secretPath = config.GetSecretSourcePath(secretName)
 
-				data := map[string]string{
-					"user":     "admin",
-					"password": "redhat",
+				secret := libsecret.New(secretName, libsecret.DataString{"user": "admin", "password": "redhat"})
+				_, err := kubevirt.Client().CoreV1().Secrets(testsuite.GetTestNamespace(nil)).Create(context.Background(), secret, metav1.CreateOptions{})
+				if err != nil && !errors.IsAlreadyExists(err) {
+					panic(err)
```

Comment:
I think the reason `panic` was used is to avoid returning an error from that helper and at the same time not to use an assertion in a helper.

Therefore, it makes sense to convert this to a regular assert once it is surfaced.

Reply:
Done